packages/bff/src/config/config.ts

```diff
@@ -67,10 +67,20 @@ export const AuthorizationProcessServerConfig = z
   .object({
     AUTHORIZATION_PROCESS_URL: APIEndpoint,
     TENANT_ALLOWED_ORIGINS: z.string(),
+    SAML_AUDIENCE: z.string(),
+    PAGOPA_TENANT_ID: z.string(),
+    SAML2_CALLBACK_URL: z.string().url(),
+    SAML2_CALLBACK_ERROR_URL: z.string().url(),
```

Comment:
to keep configurations compatibility
```suggestion
    SAML_CALLBACK_URL: z.string().url(),
    SAML_CALLBACK_ERROR_URL: z.string().url(),
```